src/views/UserProfile.vue

```diff
@@ -129,50 +129,59 @@ along with this program.  If not, see <http://www.gnu.org/licenses/>.
 
             <v-layout row align-center wrap>
               <v-flex xs3>
-                <span>Colour Theme</span>
+                <span>Color Theme</span>
```

Comment:
(Controversial decision :rofl: )

Reply:
Sorry, I think I just did that automatically in one of the commits. Let me undo this one :+1: (we will probably have en_GB and en_US in translations at some point I think).